homeassistant/components/modbus/sensor.py

```diff
@@ -55,7 +56,7 @@
     DATA_TYPE_UINT,
     DEFAULT_HUB,
     DEFAULT_SCAN_INTERVAL,
-    DEFAULT_STRUCT_FORMAT,
```

Comment:
If that is no longer needed, then please remove it from const.py, and explain why change the name.